roles/openshift-applier/tasks/process-one-entry.yml

```diff
@@ -76,6 +83,7 @@
   failed_when:
   - command_result.rc != 0
   - "'AlreadyExists' not in command_result.stderr"
+  - "'NotFound' not in command_result.stderr"
```

Comment:
Any chance we get a `NotFound` when in `create` or `apply` mode? If so, this will now be silently ignored ...

Reply:
@oybed I believe the only other cases where you could get a `Not Found` would be if you happened to pass `get` or `replace` as a file_action, which we don't document as valid values.